WalletWasabi/BranchNBound/SendCoinSelector.cs

```diff
@@ -0,0 +1,253 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.BranchNBound
+{
+	public class SendCoinSelector
```

Comment:
IMO we need a better name. This sounds like the default coin selector, but it's not.